src/c-writer.cc

```diff
@@ -3380,6 +3384,71 @@ void CWriter::Write(const ConvertExpr& expr) {
   }
 }
 
+bool CWriter::MaybeWriteNoSandboxFunctionAddress(Offset instruction_offset,
+                                                 bool is64bit) {
+  int index_byte_length = is64bit ? 10 : 5;
```

Comment:
I am a bit confused by this - can you add a short explanation (or maybe link to spec) here? Does it have something to do with instruction length?

Reply:
Yeah, i32 and i64 varieties of the instructions use 5-byte var[u]int32 and 10-byte var[u]int64 respectively.
I added comments to that effect.